pkg/api/dashboard.go

```diff
@@ -326,6 +326,12 @@ func (hs *HTTPServer) deleteDashboard(c *contextmodel.ReqContext) response.Respo
 		hs.log.Error("Failed to disconnect library elements", "dashboard", dash.ID, "user", c.SignedInUser.UserID, "error", err)
 	}
 
+	// deletes all related public dashboard entities
+	err = hs.PublicDashboardsApi.PublicDashboardService.DeleteByDashboard(c.Req.Context(), dash)
```

Comment:
Now we are doing this I think we should delete the hardcoded deletion on `dashboards/database/database.go`

Reply:
WOW, cant believe I forgot to do that 😮